src/azure-cli/azure/cli/command_modules/eventgrid/custom.py

```diff
@@ -1108,7 +1137,12 @@ def cli_system_topic_event_subscription_update(
         delivery_identity_endpoint=None,
         delivery_identity_endpoint_type=None,
         deadletter_identity=None,
-        deadletter_identity_endpoint=None)
+        deadletter_identity_endpoint=None,
+        storage_queue_msg_ttl=storage_queue_msg_ttl,
+        enable_advanced_filtering_on_arrays=enable_advanced_filtering_on_arrays)
+
+    print('print systemtopic update params', file=sys.stderr)
+    print(params.filter.enable_advanced_filtering_on_arrays, file=sys.stderr)
```

Comment:
Same debug code here😀

Reply:
yes removing all print statements in eventgrid module. Thanks!